Makefile

```diff
@@ -85,14 +86,20 @@ qm_dropin_mount_bind_ttyUSB0: ##     - Creates a QM RPM sub-package to mount bin
 	tools/version-update -v ${VERSION}
 	$(MAKE) VERSION=${VERSION} rpm
 
+.PHONY: qm_dropin_mount_bind_sound
+qm_dropin_mount_bind_sound: ##       - Creates a QM RPM sub-package to mount bind /dev/snd in the nested containers
+	sed -i 's/%define enable_qm_mount_bind_sound 0/%define enable_qm_mount_bind_sound 1/' ${SPECFILE}
+	tools/version-update -v ${VERSION}
```

Comment:
If the intention is to allow `make` to update the version, we could move it to the `rpm` target instead. So that it does not need to be explicitly added for each other (subpackage) target.

Reply:
@aesteve-rh agreed. let me know if you would like to send a patch.

Reply:
Can do.